src/Illuminate/Foundation/Exceptions/Renderer/Listener.php

```diff
@@ -44,7 +44,7 @@ public function registerListeners(Dispatcher $events)
     /**
      * Returns the queries that have been executed.
      *
-     * @return array<int, array{sql: string, time: float}>
+     * @return array<int, array{connectionName: string, time: float, sql: string, bindings: array}>
```

Comment:
This can be even done to the second array:

```suggestion
     * @return array<int, array{connectionName: string, time: float, sql: string, bindings: array<string, mixed>}>
```